src/nasuta-sebastian/Fsd.Sebastian.Client/Fsd.Sebastian.Client.Ex2/helpers.js

```diff
@@ -0,0 +1,46 @@
+var app = app || {};
```

Comment:
ja bym ten plik nazwał inaczej niż helpers. To nie jest dobra praktyka - lepiej wiedzieć za co jest odpowiedzialny. U Ciebie wygląda raczej na 'viewManager' lub 'view' lub 'appView' lub 'viewBuilder' itp